ethrpc/methods.go

```diff
@@ -282,6 +283,55 @@ func PendingTransactionCount() CallBuilder[uint] {
 	}
 }
 
+func ContractQuery(contractAddress common.Address, inputAbiExpr, outputAbiExpr string, args interface{}) (CallBuilder[[]string], error) {
```

Comment:
can we rename this to `ContractQueryBuilder` and leave the other `ContractQuery` method as it was...? this will avoid having to create breaking changes to all other places which use `ContractQuery`